src/components/SlateEditor/plugins/embed/EditImage.jsx

```diff
@@ -158,41 +150,17 @@ class EditImage extends Component {
                 imageUpdates={imageUpdates}
                 {...this.props}
               />
-              <StyledInputWrapper>
-                <Input
-                  name="caption"
-                  label={`${t('form.image.caption.label')}:`}
-                  value={caption}
-                  onChange={e => this.onEdit('caption', e.target.value)}
-                  container="div"
-                  type="text"
-                  autoExpand
-                  placeholder={t('form.image.caption.placeholder')}
-                  white
-                />
-                <Input
-                  name="alt"
-                  label={`${t('form.image.alt.label')}:`}
-                  value={alt}
-                  onChange={e => this.onEdit('alt', e.target.value)}
-                  container="div"
-                  type="text"
-                  autoExpand
-                  placeholder={t('form.image.alt.placeholder')}
-                  white
-                  warningText={
-                    !submitted && isEmpty(alt) ? t('form.image.alt.noText') : ''
-                  }
-                />
-                <StyledButtonWrapper paddingLeft>
-                  <Button onClick={this.onAbort} outline>
-                    {t('form.abort')}
-                  </Button>
-                  <Button disabled={!madeChanges} onClick={this.onSave}>
-                    {t('form.image.save')}
-                  </Button>
-                </StyledButtonWrapper>
-              </StyledInputWrapper>
+              <FigureInput
+                t={t}
```

Comment:
Vits å sende ned t? Og ikke bruker ndla-i18n?

Reply:
Fordi jest feilet for nestede komponenter som tok inn ndla-i18n